app/controllers/users/password_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Users
+  class PasswordController < ApplicationController
+   before_action :authenticated?, only: [:edit, :update]
+ 
+   def edit
+     if current_user
+       @user = current_user
+     else; @user = User.find(params["format"])
```

Comment:
This is another example of the else thing that seems strange to me.

I can understand what is going on and it doesn't really detract from the readability, but I don't think this is the typical ruby style.